appcelerator-archive/amp
# Adding a since parameter to amp logs in order to retrieve more entries

Fix #1523 

## How to test
```
$ ampmake build
$ amp cluster create
$ amp user signup --name user --email user@email.amp --password password
$ amp logs -i --since 100 amp_amplifier
$ amp logs -i --since 100 amp_amplifier
[user user in organization default @ localhost:50101]
         amp_amplifier.0 | {"level":"info","msg":"amplifier (server version: 0.13.0-dev, build: 80d068ea)\n","time":"2017-07-12T19:21:02Z"}
         amp_amplifier.0 | {"level":"info","msg":"Configuration file successfully loaded","time":"2017-07-12T19:21:02Z"}
         amp_amplifier.0 | {"level":"info","msg":"Version: 0.13.0-dev\nBuild: 80d068ea\nPort: :50101\nH1Port: :5100\nEtcdEndpoints: [http://etcd:2379]\nElasticsearchURL: http://elasticsearch:9200\nNatsURL: nats://nats:4222\nDockerURL: unix:///var/run/docker.sock\nDockerVersion: 1.27\nRegistration: none\nNotifications: false\n","time":"2017-07-12T19:21:02Z"}
         amp_amplifier.0 | {"level":"info","msg":"Connecting to etcd at http://etcd:2379","time":"2017-07-12T19:21:02Z"}
         amp_amplifier.0 | {"level":"info","msg":"Connected to etcd at http://etcd:2379","time":"2017-07-12T19:21:02Z"}
         amp_amplifier.0 | {"level":"info","msg":"Successfully created initial super user","time":"2017-07-12T19:21:02Z"}
         amp_amplifier.0 | {"level":"info","msg":"Successfully created super organization","time":"2017-07-12T19:21:02Z"}
         amp_amplifier.0 | {"level":"info","msg":"Successfully created default organization","time":"2017-07-12T19:21:02Z"}
         amp_amplifier.0 | {"level":"info","msg":"Listening on port: 50101","time":"2017-07-12T19:21:02Z"}
         amp_amplifier.0 | {"level":"info","msg":"Listening HTTP/1 on port: 5100","time":"2017-07-12T19:21:02Z"}
         amp_amplifier.0 | {"level":"info","msg":"Successfully created user user","time":"2017-07-12T19:21:14Z"}
         amp_amplifier.0 | {"level":"info","msg":"Successfully logged user in user","time":"2017-07-12T19:21:14Z"}
         amp_amplifier.0 | {"level":"info","msg":"rpc-logs: Get size:1000 service:\"amp_amplifier\" includeAmpLogs:true since:100 ","time":"2017-07-12T19:21:20Z"}
         amp_amplifier.0 | {"level":"info","msg":"rpc-logs: Get successful, returned 12 entries\n","time":"2017-07-12T19:21:20Z"}
```
Make sure that you see **since:100** in the request.
